group/init/schema.js

```diff
@@ -2,19 +2,17 @@
 //
 // SPDX-License-Identifier: LGPL-3.0-only
 
-const { tangle } = require('../definitions')
+const { groupKey, tangle } = require('../definitions')
 
 module.exports = {
   type: 'object',
-  required: [
-    'type',
-    'tangles'
-  ],
+  required: ['type', 'groupKey', 'tangles'],
```

Comment:
@Powersource I commented on this, but must have not been published for some reason.
This is a breaking change AND we need to conserve the last definition of the schema for legacy support purposes.
I will open an new PR